### ISSUE TITLE
"Failed to load board configuration. Invalid Data" error

### ISSUE THREAD

**Issue by strotee** [ccd0/4chan-x#2244]
I get the error "Failed to load board configuration. Invalid Data" and 4chan fails to load (another error quickly loads but disappears before I'm able to copy it, though I took a screenshot). Disabling the extension allows the page to load properly.

https://i.imgur.com/13kGfeM.png

This even occurred in a VM with a fresh installation of Win10 1809 + Chrome + 2 extensions (UBlock Origin and 4chanX). Everything was working earlier today 3/12/2019.

I also took a screenshot of the errors I'm getting in Chrome:
https://i.imgur.com/KiUB3k0.png

**### Update**: Oddly, it started working on its own. By "working", I mean, I can post but everything the poster below me stated is true.

**Comment by FuZ38** [comment]
Same here, it started today but I only have the "Failed to load board configuration. Invalid Data", the board loads fine but I don't have access to some 4chan x features.

-When I reply to a thread the quick reply box doesn't go away and the thread doesn't automatically refresh.
-I see my replies but can't see if they're mine (no blue thing on the left of my message and can't get (You)s), also the threads don't automatically appear in the thread watcher when I post in a thread so I don't get notifications when someone reply to my post.
-The update button at the bottom of a thread doesn't work and I have to manually refresh the thread with my browser.

**Comment by ccd0** [comment]
4chan has been having some technical issues today apparently.

**Comment by FuZ38** [comment]
I just realized I had both the tampermonkey script and extension from the Chrome store installed at the same time.
I uninstalled the extension and now it's working.

**Comment by mccpc** [comment]
I have the same issues what FuZ mentioned (error message, watcher not working, no yous, no auto refresh). Only have Chrome/Ublock/4chanx installed, never had anything else. It happens on every board. As far as I can remember there was a chrome update yesterday but nothing else was out of ordinary.

**Comment by mccpc** [comment]
Here is the exact same problem with a solution... https://github.com/ccd0/4chan-x/issues/2228
As soon as I disabled network service it works. The latest Chrome update must have set it on or something.
Maybe just me but it seems it loads the pages slightly slower.
What I tested:
- no error message anywhere (before I got it on every refresh)
- thread watcher (auto) updates as it should and notices (You) - works properly in general
- thread (auto) updates and shows everything properly including (You)
- reply adds thread to watcher properly
